packages/react-core/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@aws-amplify/ui-react-core",
+  "version": "0.0.1",
```

Comment:
Just confirming, we're good with 0.0.1 as our starting version number?

I'm not concerned because this package is internal, but just making sure.

Reply:
Resolving this, changeset will bump to 1.0.0 for us